common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -0,0 +1,19 @@
+@()(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+
+<div class="new-header__edition-picker">
+    @fragments.inlineSvg("world", "icon")
+
+    <select onchange="location = this.value;">
```

Comment:
Thinking re:accessibility - perhaps this should be a little more progressively enhanced?
the best I could think of is a list of links that then gets converted into a `select`.